lib/libc/minimal/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@
 zephyr_system_include_directories(include)
 
 zephyr_library()
+
+set(GEN_DIR ${ZEPHYR_BINARY_DIR}/misc/generated)
+zephyr_library_include_directories(${GEN_DIR})
+
+set(STRERROR_TABLE_H ${GEN_DIR}/libc/minimal/strerror_table.h)
```

Comment:
Should this be using `include/generated` as do others?

Reply:
@stephanosio  - Done!